src/screens/FeedScreen/ContentLink.js

```diff
@@ -29,7 +29,8 @@ const ContentLink = ({
   messageContainerStyle = {},
   topics = [],
   isPostDetail,
-  contentHeight
+  contentHeight,
+  hasComment
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The new prop `hasComment` has been added to the component. Ensure that this prop is being correctly passed from the parent component and that it accurately represents whether the content has a comment or not.



<!-- This is an auto-generated comment by OSS CodeRabbit -->